src/engines/kotor/gui/gui.cpp

```diff
@@ -279,6 +279,16 @@ void GUI::addBackground(const Common::UString &background) {
 		_background->setType(background);
 }
 
+void GUI::setCheckBoxState(const char* tag, bool state) {
```

Comment:
Please make sure you put the `*` at the right place, with the name instead of the tag, all over this commit too